app/javascript/components/admin/AuditModal.js

```diff
@@ -126,9 +129,12 @@ class AuditModal extends React.Component {
 
   /**
    * Formatting method for displaying each audit action in the table.
-   * @param {Object} change - audit to display holding the updated element ({String} name) and the before & after values ({Array} details)
+   * @param {*} change - Audit to display holding the updated element ({String} name) and the before & after values ({Array} details)
+   * @param {Object} rowData - Data in the row this button will be created in
+   * @param {Number} colData - Column data for the col this filter is a part of.
    */
-  formatChange = change => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Was eslint complaining about this?

Reply:
Yes, this function is called by the CustomTable filter which passes the same data to everyone but only some params are used by most. I'm likely going to add a change that packages it all into one object to avoid having to add this eslint line.

Reply:
This has since been removed: https://github.com/SaraAlert/SaraAlert/pull/604/commits/30d4fbeb075c861ae8a66567d663f31a9b23ee50